packages/repository/src/types/project.types.ts

```diff
@@ -1,6 +1,6 @@
 import { Address, ChainId } from "@grants-stack-indexer/shared";
 
-export type ProjectType = "canonical" | "linked";
+export type ProjectType = "canonical" | "linked" | "unknown";
```

Comment:
i thought we wouldn't use "unknown" as value

Reply:
good catch